.github/workflows/codegen.yaml

```diff
@@ -20,88 +20,19 @@ jobs:
           script: |
             console.log('splitting service names list into batches')
             const services = ${{ needs.discovery.outputs.services }}
+            const hour = new Date().getHours()
+            const MAX_BATCH_SIZE = 100
```

Comment:
Can we add a final step in the workflow that will fail if we have over 300 libraries so that we will be notified when this happen in the future and can adjust the configuration?

Reply:
I added a check for this case. The limit is actually 400 libs (00-03 inclusive)